src/lightning_app/utilities/warnings.py

```diff
@@ -1,4 +1,4 @@
-# Copyright The PyTorch Lightning team.
+# Copyright The Lightning team.
```

Comment:
You might want to update this also under src/pytorch_lightning and fabric

Reply:
yes, lest just do it in separate PR :)